polartools/hkl_utils.py

```diff
@@ -0,0 +1,791 @@
+# from instrument.collection import diffract
+# from gi.repository import Hkl as libhkl
+from hkl import cahkl
+
+# import pyRestTable
+from hkl.diffract import Diffractometer
+from hkl.util import Lattice
+from bluesky import RunEngine, RunEngineInterrupted
+import bluesky.plans as bp
+import bluesky.plan_stubs as bps
+from hkl.user import calc_UB, _check_geom_selected, _geom_
+
+from instrument.framework import RE
```

Comment:
This is a problem. The functions that need RE may need to go to polar-instrument.